tracer/reporter.go

```diff
@@ -0,0 +1,92 @@
+package tracer
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	wf "github.com/wavefronthq/wavefront-sdk-go/senders"
+)
+
+type ApplicationTags struct {
+	application string
+	service     string
+	Cluster     string
+	Shard       string
+	tags        map[string]string
+}
+
+func NewApplicationTags(app, serv string) ApplicationTags {
+	return ApplicationTags{
+		application: app,
+		service:     serv,
+		tags:        make(map[string]string, 0),
+	}
+}
+
+// WavefrontSpanReporter implements the wavefront.Recorder interface.
+type WavefrontSpanReporter struct {
+	Source      string
+	sender      wf.Sender
+	application ApplicationTags
+}
+
+// NewSpanReporter returns a WavefrontSpanReporter for the given `sender`.
+func NewSpanReporter(sender wf.Sender, application ApplicationTags) *WavefrontSpanReporter {
```

Comment:
No need to export `Source`. Let's take source as an argument here, and if empty we will default to hostname.

Reply:
Please also update the readme accordingly.